docs/tutorials/basic/symbol.md

```diff
@@ -201,12 +350,25 @@ ex_gpu.forward()
 ex_gpu.outputs[0].asnumpy()
 ```
 
+We can also use `eval` method to evaluate the symbol. It combines calls to `bind`
+and `forward` methods.
+
+```python
+ex = c.eval(ctx = mx.cpu(), a = mx.nd.ones([2,3]), b = mx.nd.ones([2,3]))
+print 'number of outputs = %d\nthe first output = \n%s' % (
+            len(ex), ex[0].asnumpy())
+```
+
+For neural nets, a more commonly used pattern is ```simple_bind```, which
+creates all of the argument arrays for you. Then you can call ```forward```,
+and ```backward``` (if the gradient is needed) to get the gradient.
+
 ### Load and Save
 
-Similar to NDArray, we can either serialize a `Symbol` object by using `pickle`,
-or use `save` and `load` directly. Different to the binary format chosen by
-`NDArray`, `Symbol` uses the more readable json format for serialization. The
-`tojson` method returns the json string.
+Similar to `NDArray`, we can either serialize a `Symbol` object by using `pickle`,
```

Comment:
We need to consider introducing the concept of *Graph*. 
Logically Symbols correspond to NDArray; they both represent a tensor. They are both outputs/inputs of operators. Graph is composed by chaining operators. We don't have a Graph object in frontend (but we do have graph in backend) Graphs are implicitly represented by output symbols.